app/controllers/api/v1/contacts_controller.rb

```diff
@@ -26,33 +26,31 @@ def index
         end
       end
 
-			def show
-        company = @current_user.companies.find_by(id: params[:id])
-        authorize company
-
-        if company
-          contacts = company.contacts
-          render json: { company: CompanySerializer.new(company), contacts: ContactSerializer.new(contacts) }
+      def create 
+        authorize Contact
+        if params[:company_id] 
```

Comment:
Similar to the company controller, this could potentially be refactored into the model. While not necessary for this specific PR, something to make note of, especially if there is any more logic added to the action at a later time.

Reply:
Yes, definitely. There is a ticket up for that, #58 in backlog.

Reply:
Ope! I'm sorry I forgot that you mentioned that in the stand up.